core/shared/src/main/scala/cats/parse/Parser.scala

```diff
@@ -516,6 +520,11 @@ sealed abstract class Parser[+A] extends Parser0[A] {
   def rep(min: Int): Parser[NonEmptyList[A]] =
     Parser.repAs(this, min = min)
 
+  def rep(min: Int, max: Int): Parser[NonEmptyList[A]] = {
+    require(min >= 1, s"min should be >= 1 but was $min")
```

Comment:
Should we move this require into repAs? Seems this call has the require but the one above does not.

I tend to say yes.

Reply:
I'll take another pass over the require's and make sure everything in the layer that calls `Impl.Rep`, so that we're sure they're called exactly once, with min/max identical to the ones the user passed.